mathematica/newhall.nb

```diff
@@ -346,7 +346,44 @@ Cell[BoxData[
    RowBox[{"NewhallC2", "[", 
     RowBox[{"degree", ",", "divisions", ",", "w"}], "]"}]}]}]], "Input",
  CellChangeTimes->{{3.640856537054928*^9, 3.6408566025854797`*^9}, 
-   3.6408567878820076`*^9}]
+   3.6408567878820076`*^9}],
+
+Cell["\<\
+This function expresses C in a way that is suitable for obtaining the \
```

Comment:
C should be in TraditionalForm. Several other instances of missing TraditionalForm.

Reply:
Done.